five/intid/keyreference.py

```diff
@@ -110,7 +110,7 @@ def wrapped_object(self):
             return self.object
         try:
             obj = self.root.unrestrictedTraverse(self.path)
-        except (NotFound, AttributeError,):
+        except (NotFound, AttributeError, KeyError):
```

Comment:
LGTM but I think this will take care of everything:
```
obj = self.root.unrestrictedTraverse(self.path, None)
if obj is None:
     return self.object
```
I would prefer this implementation because that exact same exception handling is already done in the `unrestrictedTraverse` method itself:
https://github.com/zopefoundation/Zope/blob/57da264fbcf1c0577470231208683b9d1b38de60/src/OFS/Traversable.py#L321

Reply:
Done.